frontends/ol-components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import FormControl from "@mui/material/FormControl"
+import { Select } from "../SelectField/SelectField"
+import MenuItem from "@mui/material/MenuItem"
+import type { SelectChangeEvent } from "@mui/material/Select"
+
+interface DropdownProps {
+  /**
+   * title for the dropdown
+   */
+  title: string
+  /**
+   * Key for the dropdown and base for key for dropdown options
+   */
+  keyName: string
+  /**
+   * Value of initial selection for the dropdown
+   */
+  initialValue: string | string[]
+  /**
+   * Whether the dropdown allows multiple selections
+   */
+  isMultiple: boolean
+  /**
+  The function that runs when there is a selection from the dropdown
+   */
+  onChange: (event: SelectChangeEvent) => void
+  /**
+   * The options for the dropdown
+   */
+  options: DropdownOptionProps[]
+}
+
+interface DropdownOptionProps {
+  /**
+   * value for the dropdown option
+   */
+  key: string
+  /**
+   * label for the dropdown option
+   */
+  label: string
+}
+
+const Dropdown: React.FC<DropdownProps> = ({
```

Comment:
It seems to me that "Dropdown" is mostly a simplified interface for MUI's Select component. I think that's good and convenient. My understanding of "Dropdown" though is that it's usually a menu, not something that stores a value—that's a Select.

 What would you think of calling it `SimpleSelect` to emphasize that (a) it's a Select component, (b) it's simpler, and (c) it's sort of parallel to our existing SimpleMenu (which is a simplified MUI Menu interface).